apps/docs/src/pages/components/accordion.mdx

```diff
@@ -0,0 +1,127 @@
+import { Callout } from "nextra/components";
+import {
+  accordionClasses,
+  HvAccordion,
+  HvListContainer,
+  HvListItem,
+  HvPanel,
+  HvTypography,
+} from "@hitachivantara/uikit-react-core";
+
+import { Description } from "@docs/components/component/Description";
+import { Page } from "@docs/components/component/Page";
+
+import Playground from "./Playground";
+
+import { getComponentData } from "../../utils/utils";
+
+export const getStaticProps = async ({ params }) => {
+  const meta = await getComponentData("Accordion", "core", accordionClasses);
+  return { props: { ssg: { meta: meta } } };
+};
+
+<Description />
+
+<Page>
+
+<Playground
+  Component={HvAccordion}
+  componentName="HvAccordion"
+  controls={{
+    label: {
+      type: "text",
+      defaultValue: "Analytics",
+    },
+    labelVariant: {
+      defaultValue: "label",
+    },
+    disabled: {
+      defaultValue: false,
+    },
+  }}
+  componentProps={{ style: { width: "50%" } }}
+  children={
+    <HvListContainer interactive condensed>
+      <HvListItem className="pl-32px">Views</HvListItem>
+      <HvListItem className="pl-32px">Parameters</HvListItem>
+    </HvListContainer>
+  }
+/>
+
+<Callout type="info">
+  You can use the `HvSection` component to create an accordion group that better
+  fits your needs. Please refer to the [Section
+  documentation](/components/section#multiple) for usage.
+</Callout>
+
+### Controlled
+
+You can control the state of `HvAccordion` by using the `expanded / onChange` props.
+
+```jsx live
+import { useState } from "react";
+
+export default function Demo() {
+  const [expanded, setExpanded] = useState(false);
+
+  return (
+    <div className="flex flex-col gap-2 w-full">
+      <HvButton style={{ width: 150 }} onClick={() => setExpanded((p) => !p)}>
+        Open
+      </HvButton>
+      <HvAccordion
+        expanded={expanded}
+        onChange={(e, isExpanded) => setExpanded(isExpanded)}
+        label="Analytics"
+        labelVariant="label"
+      >
+        <HvListContainer interactive condensed>
+          <HvListItem className="pl-32px">Views</HvListItem>
+          <HvListItem className="pl-32px">Parameters</HvListItem>
+        </HvListContainer>
+      </HvAccordion>
+    </div>
+  );
+}
+```
+
+### Multiple accordions
+
+If you need an accordion group, you can use multiple `HvAccordion` components.
+
+```jsx live
+<div className="w-full flex flex-col">
+  <HvTypography className="pb-4" variant="title3">
+    UI Kit
+  </HvTypography>
+
+  <HvAccordion
+    label="Components"
+    labelVariant="title4"
+    style={{ width: "50%" }}
```

Comment:
instead of having this this on each Accordion, we move it to the the parent div (eg. `w-1/2`)
```suggestion
```